applications/projects/CMakeLists.txt

```diff
@@ -16,7 +16,16 @@ endif()
 
 sofa_add_application(GenerateRigid GenerateRigid)
 sofa_add_application(meshconv meshconv OFF)
-sofa_add_application(SofaPhysicsAPI SofaPhysicsAPI)
+
+## Glut GUI
+find_package(GLUT)
+if(GLUT_FOUND)
+	sofa_add_application(SofaPhysicsAPI SofaPhysicsAPI)
```

Comment:
Would be better to add every app disabled and let them fail when running `find_package(GLUT REQUIRED)` if enabled, no?  
I don't think this CMakeLists should be smartly checking `GLUT_FOUND`. It's not its dependency.

Reply:
I was wondering the same, but I was lazy to add the check in all the projects needing glut.
But I guess you are right so I will add it in those projects directly